_posts/2019-02-15-introducing-aquery.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: posts
+title: Introducing Bazel Aquery
+authors:
+ - joeleba
+---
+
+_tl;dr: [`bazel aquery`](https://docs.bazel.build/versions/master/aquery.html) is a new bazel command that queries the action graph, and thus allows you to gain insights about the actions executed in a build (inputs, outputs, command line, …).
+`aquery`’s API is now stable and supported by the Bazel team._
+
+## Why `bazel aquery`?
+
+_"What was the command line that produced this output file?"_
+
+_"Did the new change in my rule implementation affect the current list of actions?"_
+
+_"What was the exact command line used to run action X?"_
```

Comment:
Is this example similar to the first one? Could you more precisely show what's different between them?

Reply:
My bad. Replaced it with a new one.